crux-core/src/crux/db.clj

```diff
@@ -66,4 +67,17 @@
 
 (defprotocol DocumentStore
   (submit-docs [this id-and-docs])
-  (fetch-docs [this ids]))
+  (-fetch-docs [this ids]))
+
+(defn fetch-docs [document-store ids]
+  (let [ids (set ids)]
+    (loop [indexed {}]
```

Comment:
Mini nitpick, the name `indexed` here doesn't feel clear, what is indexed here? Is this more simply `docs` or something similar? Assume this might been moved from a context where that name made more intuitive sense.

Reply:
Yep - it made sense in its original context but now doesn't, will change :)